Bill3.java

```diff
@@ -1,8 +1,14 @@
 // Splits a restaurant bill evenly among three diners.
 public class Bill3 {
 	public static void main(String[] args) {
+		
 		// To get you started, here is the first line in the program:
-	    String name1 = args[0];
-	    // Replace this comment with the rest of your code   
+		int N = Integer.parseInt(args[3]);
+		 String name1 = args[0];
+		String name2 = args[1];
+		String name3 = args[2];
+		double i = (double)N/3; 
```

Comment:
it is rather better to work with int whenever possible. 
The way wrote it first casts N into double then divide by 3.